website/contributing/release-branch-cut-and-rc0.md

```diff
@@ -39,6 +39,9 @@ Documents in this section go over steps to run different types of React Native r
 
 - Add and commit the extra file that got created at `sdks/hermes/.hermesversion`.
 
+- Update packages in the monorepo by running `npm run bump-all-updated-packages`. All the package bumps should be a patch. Read more about the script and how they work [here](./release-updating-packages).
```

Comment:
Can we add one more sentence on why we need to do it?

Maybe something like "We need to publish the latest available changes in our code, so they will be included in a new RC"